data/RI/incentives.json

```diff
@@ -245,6 +245,54 @@
       "es": "Un reembolso adicional de $1,500 por un vehículo eléctrico usado para los calificadores de LIHEAP. Por orden de llegada."
     }
   },
+  {
+    "id": "RI-13",
+    "authority_type": "state",
+    "authority": "ri-oer",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "ebike",
+    "program": "ri_erikaNiedowskiMemorialEBikeRebateProgram",
+    "amount": {
+      "type": "percent",
+      "number": 0.3,
+      "maximum": 350
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "low_income": "default",
```

Comment:
confirmed that the income restrictions on the [website](https://drive.ri.gov/erika-niedowski-memorial-electric-bicycle-rebate-program) are the same as the [default ones for RI](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/data/low_income_thresholds.json#L173-L187)